src/Symfony/Bundle/FrameworkBundle/Templating/Helper/AssetsHelper.php

```diff
@@ -78,7 +78,7 @@ public function getVersion($path = null, $packageName = null)
 
         // packageName is null and path not, so path is a path or a packageName
         try {
-            $package = $this->packages->getPackage($path);
+            $this->packages->getPackage($path);
```

Comment:
A while ago we standardized the exception names to only use `$e` because previously we used semantic names for some exceptions ... and was confusing. The exception meaning is provided by the type-hinted class.